src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Write your first NRQL alerts condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alerts conditions with a step-by-step user case.
+---
+
+Use our [New Relic query language (NRQL)](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alerts conditions with ease. The query defines the data that's important to you and the condition describes what you want to be notified about.
+
+Your condition only requires two attributes: a query and a threshold. The query determines what you want to be notified about and the threshold determines when you’ll be notified. 
+
+Follow these steps to write your first alerts condition using a NRQL query and a threshold. Once you're done, you'll have a working alerts condition.
+
+## Step 1: Write your query [#step1-nrql]
+
+In the example, an alerts condition is created for each host's CPU utilization (query) that will create a notification when the CPU utilization is above 70% (threshold).
+
+Determine what alerts condition you want to query before continuing. You can create an alerts condition from any valid NRQL query. Find the NRQL query builder at the top of the UI.
+
+![A screenshot showing where you can find the NRQL query builder button in New Relic One.](./images/find-builder.png 'A screenshot showing where you can find the NRQL query builder button in New Relic One.')
+
+<figcaption>
+  Click the **Query your data** button to open the query builder and start writing a NRQL query. Once you've written a valid query, you can create an alert condition from it.
```

Comment:
alerts condition, with an S. That's how we've been using it in the rest of the doc.